nifi-nar-bundles/nifi-amqp-bundle/nifi-amqp-processors/src/main/java/org/apache/nifi/amqp/processors/AbstractAMQPProcessor.java

```diff
@@ -97,6 +98,14 @@
             .required(false)
             .identifiesControllerService(SSLContextService.class)
             .build();
+    public static final PropertyDescriptor USE_CERT_AUTHENTICATION = new PropertyDescriptor.Builder()
+            .name("Cert Authentication")
```

Comment:
Could you use ``.displayName()`` for what is displayed on the UI? and ``.name`` for a more "computer-friendly" name? It's not mandatory but that's a convention we are trying to progressively adopt.